### PR TITLE
Move to latest jackson databind

### PR DESCRIPTION
**Pull request by asoldano**

### CODE REVIEW

File: bom/runtime/pom.xml [hunk 1]
```diff
@@ -68,7 +68,7 @@
            is not available in Maven fast enough: -->
         <graal-sdk.version>19.1.1</graal-sdk.version>
         <gizmo.version>1.0.0.Alpha5</gizmo.version>
-        <jackson.version>2.9.9</jackson.version>
+        <jackson.version>2.9.9.20190727</jackson.version>
```
Comment (gsmet) [inline bom/runtime/pom.xml]: Are you sure about this one? I don't see it on Central but I see a `2.9.9.2`.

https://search.maven.org/search?q=g:com.fasterxml.jackson.core%20AND%20a:jackson-databind&core=gav

**Review by gsmet — CHANGES_REQUESTED** [review]

File: bom/runtime/pom.xml (hunk 1 above)
Comment (asoldano) [reply bom/runtime/pom.xml]: http://central.maven.org/maven2/com/fasterxml/jackson/jackson-bom/2.9.9.20190727/

Comment (asoldano) [reply bom/runtime/pom.xml]: mmh... let me fix this properly, pinging you in chat

**Comment by gsmet** [comment]
I created https://github.com/FasterXML/jackson-bom/issues/23 for the `jackson-annotations` issue. Better fix it upstream.

**Review by gsmet — APPROVED** [review]

**Pull request merged**